app/models/language.rb

```diff
@@ -46,11 +46,17 @@ class Language < ApplicationRecord
 
   validates :default_language, inclusion: { in: BOOLEAN_VALUES }
 
-  # =============
-  # = Callbacks =
-  # =============
-
-  before_validation :format_abbreviation, if: :abbreviation_changed?
+  # =========================
+  # = Custom Accessor Logic =
+  # =========================
+
+  # ensure abbreviation is downcase and conforms to I18n locales
+  # TODO: evaluate the need for the LocaleService after move to Translation.io
```

Comment:
🙏 we're having so many problems with Fastgettext right now that I'm wondering if its worth moving now